src/components/GeneralInformationProp.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <el-card class="box-card" id="general-information">
+        <div slot="header" class="clearfix">
+            <span>Informações Gerais</span>
+        </div>
+
+        <table style="width:100%">
+            <tr class="text item">
+                <td><b>Autor</b></td>
+                <td>{{ proposicao.autor }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Casa de Origem</b></td>
+                <td>{{ proposicao.casa_origem }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Data de Apresentação</b></td>
+                <td>{{ proposicao.data_apresentacao }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Ementa</b></td>
+                <td>{{ proposicao.ementa }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Explicação da Ementa</b></td>
+                <td>{{ proposicao.explicacao_ementa }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Status Atual</b></td>
+                <td>{{ proposicao.status_atual }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Forma de Apreciação</b></td>
+                <td>{{ proposicao.forma_apreciacao }}</td>
+            </tr>
+            <tr class="text item">
+                <td><b>Regime de Tramitação</b></td>
+                <td>{{ proposicao.regime_tramitacao }}</td>
+            </tr>
+            <tr class="text item">
```

Comment:
Poderia utilizar o SCSS para aplicar a classe para todos os TRs

Reply:
Eu não sei se eu usaria `scoped`...
Para mim ele tem um funcionamento meio irregular e acaba dificultando depois aproveitar estilos globais...

Reply:
Para explicar melhor:

Quando eu vi esse `scoped` pela primeira vez também achei uma ótima ideia. E também achei que seria mais fácil usar direto o nome dos elementos do que ficar criando classes.
Mas depois começaram a aparecer problemas e eu não entendia o que acontecia. E tirando o `scoped` resolvia. Então fiquei meio queimado com ele. No nosso código do Hackfest acho o problema se repetiu. Eu resolvi os bugs de CSS tirando o `scoped` de tudo.

Mas pode ser desconhecimento meu sobre como ele funciona.

Dei uma pesquisada agora e achei isso:
https://stackoverflow.com/questions/45898865/how-to-correctly-use-scoped-styles-in-vuejs-single-file-components
Que parece destacar essa característica que está na documentação e eu não conhecia:

> With scoped, the parent component's styles will not leak into child components. However, a child component's root node will be affected by both the parent's scoped CSS and the child's scoped CSS. This is by design so that the parent can style the child root element for layout purposes.
( https://vue-loader.vuejs.org/guide/scoped-css.html )

Talvez tenha sido isso que gerou meus problemas com `scoped`? Ou será que ele guarda mais segredos?

De qualquer jeito, a mesma página da documentação diz isso:

> Scoped styles do not eliminate the need for classes. Due to the way browsers render various CSS selectors, p { color: red } will be many times slower when scoped (i.e. when combined with an attribute selector). If you use classes or ids instead, such as in .example { color: red }, then you virtually eliminate that performance hit. Here's a playground where you can test the differences yourself.

Mas eu concordo que usando `scoped` e não usando classes o código fica mais enxuto (e no nosso caso o impacto na performance talvez não seja perceptível).

O que acham? Tentamos usar `scoped` sem classes e mudamos depois se tivermos problemas?

Reply:
Ah, e tem a possibilidade de usarmos o CSS Modules como cita o primeiro link que passei.
(apesar de eu não entender direito o que é isso...)